btcpp_ros2_samples/README.md

```diff
@@ -0,0 +1,38 @@
+# Sample Behaviors
+
+For documentation on sample behaviors included in this package please see the BehaviorTree.CPP [ROS 2 Integration documentation](https://www.behaviortree.dev/docs/ros2_integration) .
+
+# TreeExecutionServer Documentation and Example
```

Comment:
I'm sorry I got out of sync with you on the docs. I will clean up this doc and remove duplicate info.